chatmaild/src/chatmaild/metadata.py

```diff
@@ -117,7 +118,25 @@ def handle_dovecot_request(msg, transactions, notifier, metadata, iroh_relay=Non
         if keyname[0] == "priv" and keyname[2] == metadata.DEVICETOKEN_KEY:
             metadata.add_token_to_addr(addr, value)
         elif keyname[0] == "priv" and keyname[2] == "messagenew":
-            notifier.new_message_for_addr(addr, metadata)
+            proc = subprocess.run(
+                [
+                    "doveadm",
```

Comment:
creating a new subprocess for each incoming message is probably expensive but then the mime-parser of doveadm might be faster than python's.  However, it may make sense to find out the location of `doveadm` and specify an absolute path here.